sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  extension:
+  - added new functionality to update arguments and ND-range sizes of kernel
+    nodes within a graph. intel/llvm#12486
+  - clarified `enable_profiling` property in `command_graph::finalize`
+    intel/llvm#14067
+  - clarified how graph edges are recorded in a graph with in-order queues.
+    intel/llvm#12916
+  - clarified how code within command-group functions is handled.
+    intel/llvm#13699
+  - clarified interaction between queue properties and graphs. intel/llvm#13681
+  - added `enable_profiling` property. intel/llvm#13088
+- Updated
+  [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc)
+  extension:
+  - added support external semaphores that can take value. intel/llvm#13860
+  - added support for copying `image_mem_handle` between devices via
+    `ext_oneapi_copy`. intel/llvm#12449
+  - added `fetch_image` overload which accepts sampled image and coordinates.
+    intel/llvm#12447
+  - added type hint template argument to `fetch_cubemap` and `sample_cubemap`
+    APIs. intel/llvm#13742
+
+### SYCLcompat
+
+- Added non-`const` `image2d_max` and `image3d_max` getters. intel/llvm#14138
+- Introduced versioning scheme for the library. intel/llvm#14457
+- Enhanced  masked shuffle functions `select_from_sub_group`,
+  `shift_sub_group_left`, `shift_sub_group_right` and `permute_sub_group_by_xor`
+  to support CUDA devices. intel/llvm#13363
+- Restricted `memory_order` argument of `atomic_ref` passed to
+  `experimental::nd_range_barrier` to match supported on a device.
+  intel/llvm#12974 intel/llvm#13641
+
+## Bug Fixes
+
+### SYCL Compiler
+
+- Fixed that using `-fsycl-link-targets` flag would inadvertently trigger some
+  additional device code linking steps. intel/llvm#13004
+- Fixed a bug that when AOT-compiling for Intel GPUs the compiler would pass
+  some PVC-specific flags even if target device is not a PVC. intel/llvm#13794
+- Fixed a bug with incorrect file extensions being emitted in AOT compilation
+  when `--save-temps` is used. intel/llvm#14214
+- Made `-fsycl-add-default-spec-consts-image` available with `clang-cl`. intel/llvm#13168
+- Fixed an issue where performing separate compilation and linking with
+  `-fsycl-link` would result in "number of output files and targets should match
+  in unbundling mode" error emitted by the compiler during link step. intel/llvm#13002
+- Fixed a bug in address sanitizer which may lead to crashes when an application
+  is launched on OpenCL CPU device. intel/llvm#13262
+- Fixed a bug where calling certain built-in math functions that accepts
+  pointers (like `fract`, `frexp`, `modf`, etc.) and passing pointer in the
+  generic address space there would not compile for AMD devices.
```

Comment:
```suggestion
- Fixed a bug where calling certain built-in math functions that accept
  pointers (like `fract`, `frexp`, `modf`, etc.) and passing pointers in the
  generic address space would not compile for AMD devices.
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706